ReactAndroid/src/main/java/com/facebook/react/modules/clipboard/ClipboardModule.java

```diff
@@ -43,28 +44,42 @@ public String getName() {
   }
 
   private ClipboardManager getClipboardService() {
-    ReactApplicationContext reactContext = getReactApplicationContext(); 
+    ReactApplicationContext reactContext = getReactApplicationContext();
     return (ClipboardManager) reactContext.getSystemService(reactContext.CLIPBOARD_SERVICE);
   }
 
+  private String getStringInternal(){
+    ClipboardManager clipboard = getClipboardService();
+    ClipData clipData = clipboard.getPrimaryClip();
+    if (clipData == null) {
+      return "";
+    }
+    if (clipData.getItemCount() >= 1) {
+      ClipData.Item firstItem = clipboard.getPrimaryClip().getItemAt(0);
+      return "" + firstItem.getText();
+    } else {
+      return "";
+    }
+  }
+
   @ReactMethod
   public void getString(Callback cb) {
```

Comment:
Can we remove this from native and move the backward compatibility to JavaScript? When we deprecate and remove this, it'll be much easier.